com.unity.render-pipelines.universal/dummy.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class dummy : MonoBehaviour

```

Comment:
What is the point of this?

Reply:
It's not used by anything - I guess it was at some point. I've deleted the file.